py3dtilers/CityTiler/citym_building.py

```diff
@@ -71,7 +74,7 @@ def sql_query_objects(buildings):
         return query
 
     @staticmethod
-    def sql_query_geometries(offset, buildings_ids_arg):
+    def sql_query_geometries(buildings_ids_arg, split_surfaces=False):
         """
         :param offset: the offset (a 3D "vector" of floats) by which the
```

Comment:
The docstring doesn't match anymore. What is the split_surfaces argument done for ?